### PR TITLE
RA-1592 : Boostrap as a foundation for Reference Application UI Project.

### PR DESCRIPTION
**Pull request by Akayeshmantha**
**This PR contains changes for the GSoC Project Bootstrap as a foundation for Ref App UI project.
It contains the following changes up to now**

Account.scss css changes.
PersonDetails.gsp page bootstrap changes.
providerDetails.gsp fragment bootstrap changes.
userDetails.gsp fragment bootstrap changes.
list.gsp page bootstrap changes.
ManageLocationAttributeTypes.gsp page bootstrap changes.
ManageLocations.gsp page bootstrap changes.
ManagePrivileges.gsp page bootstrap changes.
Account.gsp page bootstrap changes.
ManageAccounts.gsp bootstrap changes.
Bootstrap integration(Currently works with internet only).

**This PR will have more commit's when the project is going forward.**

Ticket ID :- RA-1952

### CODE REVIEW

**Comment by HerbertYiga** [comment]
@Akayeshmantha   some failing checks here!!!

File: omod/src/main/webapp/fragments/systemadmin/accounts/personDetails.gsp [hunk 1]
```diff
@@ -23,7 +23,7 @@
     <div class="adminui-form-person-details-panel">
 <% } %>
         <fieldset class="<% if(createAccount) { %>adminui-account-fieldset<% } else { %>adminui-form-person-details-panel<% } %>">
-        <legend><b>${ ui.message("adminui.person.details") }</b></legend>
+        <legend class="w-auto"><b>${ ui.message("adminui.person.details") }</b></legend>
```
Comment (suthagar23) [inline omod/src/main/webapp/fragments/systemadmin/accounts/personDetails.gsp]: what does it mean by `w-auto`? is it `width-auto`?

Comment (Akayeshmantha) [reply omod/src/main/webapp/fragments/systemadmin/accounts/personDetails.gsp]: Hi @suthagar23 yes these are predefined tags now in bootstrap.

For more ref.: -
https://getbootstrap.com/docs/4.1/utilities/sizing/

Comment (ssmusoke) [reply omod/src/main/webapp/fragments/systemadmin/accounts/personDetails.gsp]: @suthagar23 Is this good to go?

File: omod/src/main/compass/sass/systemadmin/account.scss [hunk 2]
```diff
@@ -8,10 +8,10 @@
   width: 500px;
 }
 
-.adminui-account-section {
-   width: 980px;
-}
+// .adminui-account-section {
```
Comment (ssmusoke) [inline omod/src/main/compass/sass/systemadmin/account.scss]: @Akayeshmantha delete unused code instead of commenting it out

**Pull request merged**